template-contributing-guide.md

```diff
@@ -0,0 +1,341 @@
+
+
+### Required deliverables
+
+Currently, for a template set to be considered complete, each template set should have these documentation deliverables:
+
+<table>
+  <tr>
+    <th>Deliverable</th>
+    <th>Description and purpose</th>
+    <th>Required?</th>
+  </tr>
+  <tr>
+    <td>Raw template</td>
+    <td>The raw template is the basic template for the type of document you are creating. It provides a rough outline of the suggested content and a few embedded writing tips for how to fill in the different sections of the template.</td>
+    <td>Yes, required</td>
+  </tr>
+  <tr>
+    <td>Template guide</td>
+    <td>This guide provides a deeper explanation of how to fill in the raw template. It provides a lightweight introduction to the purpose of this documentation and explains how to fill in each section of the document. Additional content may be included as needed.</td>
+    <td>Yes, required</td>
+  </tr>
+  <tr>
+    <td>Before you start guide</td>
+    <td>This optional guide is designed to help template users understand some of the key research, brainstorming, or pre-writing steps that they might need to do before adding this type of document to a guide. If this documentation type has certain key decisions that need to be made with stakeholders or if it has a dependency on some other type of document or process, that should be mentioned in this guide.</td>
+    <td>No, optional</td>
+  </tr>
+  <tr>
+    <td>Template example</td>
+    <td>The Good Docs Project is in the process of designing a fake documentation project that can provide examples of our templates in action. Until this project is ready, it is optional to provide an example of the template. However, if you want to create a fake example as part of your template set, you may do so.</td>
+    <td>No, optional</td>
+  </tr>
+</table>
+
+For more detailed information about each deliverable, see [Template Deliverables](template-deliverables.md).
+
+
+## Overview of the template writing phases
+Contributing a template project to our repository has five phases, as summarized in this table:
+
+<table>
+  <tr>
+    <th>Phase</th>
+    <th>Your goals</th>
+    <th>Resources to help you</th>
+  </tr>
+  <tr>
+    <td>1 - Join the community</td>
+    <td><ul>
+          <li>Join our project’s communication channels.</li>
+          <li>Decide which initiative you’d like to work on based on your interests and experience.</li>
+        </ul>
+    </td>
+    <td><ul>
+          <li>The Good Docs Project welcoming committee</li>
+          <li>Working groups</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>2 - Adopt a template</td>
+    <td><ul>
+          <li>Join a template writing working group led by a template mentor.</li>
+          <li>Work with a template mentor and your working group to decide which template you’ll work on.</li>
+          <li>Assign yourself to the corresponding issue for that template.</li>

```

Comment:
As it stands right now I'd expect templates to be represented by epics, either release-specific or otherwise. We may need to adjust this wording accordingly.

Reply:
Agreed. Do we need to resolve this before the RFC is adopted or can this be revised after the fact?

Reply:
I'm not going to take action on this comment because it's out of the scope of this RFC. Creating the template project management system will be a separate RFC.